microsoft/PowerToys
# [VCM] Track newly added microphones when [All] is selected in the set…

…tings

## Summary of the Pull Request

**What is this about:**

**What is included in the PR:** 

**How does someone test / validate:** 
- set tracked mic to [All] on the VCM settings page
- unplug your usb mic
- press win+shift+a to mute all of your mics
- plug your usb mic back
- control panel -> sound -> recording -> open properties of the just plugged mic -> levels -> observe the muted icon
## Quality Checklist

- [x] **Linked issue:** #10881

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
